views/use_the_data.erb

```diff
@@ -0,0 +1,256 @@
+			the most recent versions of each legislature within those countries, and the
+			paths you need to <em>find</em> them. From <code>countries.json</code>, you
+			can construct the URLs of the data you want.
+		</p>
+		<p>
+			Every time we change any of the data, we update <code>countries.json</code>.
+			See <a href="repo_structure.html">getting the most recent data</a>
+			for details about using this crucial file.
+		</p>
+		<h2>
+			Four policies for <em>getting the data</em>
+		</h2>
+		<p>
+			You&rsquo;re going to write the code. All you need is the data.
+		</p>
+		<p>
+			But before you go any further, we want you to think about
+			<em>how</em> you&rsquo;re going to be getting it. We deliberately
+			<em>don&rsquo;t</em> provide an API &mdash; instead, we make the data
+			available and encourage you, here, to adopt a good policy on how and when
+			you actually go about getting it. There are four approaches: we encourage
+      everyone to consider using the last one (event-driven):
+		</p>
+    <ul>
+      <li><a href="#download-it-once">download it once, never worry again</a></li>
+      <li><a href="#download-it-again">download it and make it possible to download it again</a></li>
+      <li><a href="#download-it-regularly">download it regularly</a></li>
+      <li><a href="#download-triggered-by-event">download it whenever it changes: event-driven</a> (recommended)</li>
+    </ul>
+    <a name="download-it-once"></a>
+		<h3>
+			Download it once, never worry again
+		</h3>
+		<p>
+			If you&rsquo;re doing a one-off &mdash; maybe building an infographic or
+			plotting a graph for a printed magazine or granite sculpture &mdash; this might
+			work for you. But we don&rsquo;t recommend this because <em>political data
+			changes</em>. Elections happen, politicians die, new richer data becomes
+			available, errors get corrected.
+		</p>
+		<p>
+			So if you&rsquo;re just going to download it once and never change it
+			again, make sure that&rsquo;s a deliberate decision with good reason.
+			Alternatively, use one of the following approaches.
+		</p>
+    <a name="download-it-again"></a>
+		<h3>
+			Download it and make it possible to download it again
+		</h3>
+		<p>
+			So if you are sure you want to just download the data and maybe
+			stick it into your own database, or creating static pages from the file
+			you&rsquo;ve downloaded &mdash; make sure you do it in a way you can
+			reproduce. That is, a week or a month later, if you download it again, you
+			can run the same process.
+		</p>
+		<p>
+			It depends on your project, but really we&rsquo;re suggesting you
+			write an import or update script that populates your database; or make
+			sure your code can run from scratch against a fresh download. This means you
+			need to consider things like: what happens if a politician&rsquo;s name changes?
+			Someone is removed? A whole new government is elected? Make sure your
+			application can handle this change.
+		</p>
```

Comment:
We probably need to say something about EveryPolitician IDs here — though we have the slight problem that that's about to change quite radically. However, whether on the current or hopefully-about-to-become-current approach, it's something people will need to be able to handle — either IDs aren't stable at all, or they'll hopefully be mostly stable, but will still be subject to change (e.g. if we discover two records with different IDs are really the same person and need to merge them.)